contracts/components/_old/dispatch/Dispatch_0_1_4.sol

```diff
@@ -0,0 +1,4319 @@
+// Sources flattened with hardhat v2.9.9 https://hardhat.org
```

Comment:
[The process is here](https://github.com/forta-network/forta-contracts#development-of-upgrades) but it should be expanded on the file generation step.

The files are generated following this process: 
1. hardhat flattener
2. Good ol' manual removing of repeated `pragma` statements, redundant licensing...
3. Renaming the top level file so it does not generates conflicts in the tests.

The purpose is testing the storage layout for the upgrades. The truth is this process is a PITA, I didn't fully script it because I have the muscle memory down by this point, did not want to go into forking the flattener and scripting text modification with regex usually ends up being a time sink.

This was needed before `upgrades-plugin` checked for renaming and `__gap` array modification between upgrades, but maybe now the process of upgrade testing is obsolete.

In any case I was thinking on just testing the last upgrade from now on, because every file adds to compilation and test execution time, and they are big and ugly.

What do you think?

Reply:
>but maybe now the process of upgrade testing is obsolete

No, I think this is something we should continue to do. At least some kind of smoke test. But there is no need to keep the source code and keep compiling them all the time. We can just keep the JSON artifact.